application/src/ext-test/java/org/opentripplanner/ext/flex/FlexIntegrationTestData.java

```diff
@@ -59,4 +59,6 @@ private static TestOtpModel buildFlexGraph(File file) {
     assertTrue(timetableRepository.hasFlexTrips());
     return new TestOtpModel(graph, timetableRepository);
   }
+
+  private FlexIntegrationTestData() {}
```

Comment:
What do we think about these?

Reply:
To me it doesn't add anything but perhaps there are other people that care.